ecommerce/extensions/catalogue/management/commands/migrate_course.py

```diff
@@ -45,9 +45,10 @@ def _build_lms_url(self, path):
 
     def _query_commerce_api(self):
         """Get course name and verification deadline from the Commerce API."""
-        commerce_api_client = self.site_configuration.commerce_api_client
+        api_client = self.site_configuration.oauth_api_client
+        commerce_url = urljoin(self.site_configuration.commerce_api_url, f"/courses/{self.course.id}")
```

Comment:
Here we will have an incorrect url, left slash from the url param should be cleaned.